packages/filesystem/src/browser/file-dialog/file-dialog.ts

```diff
@@ -218,3 +276,84 @@ export class FileDialog extends AbstractDialog<MaybeArray<FileStatNode>> {
     }
 
 }
+
+@injectable()
+export class SaveFileDialog extends FileDialog<URI | undefined> {
+
+    fileNameField: HTMLInputElement;
```

Comment:
`protected fileNameField: HTMLInputElement | undefined` - since it is internal and not create in the constructor

Reply:
Reworked. Thanks!